src/webapps/azext_webapps/__init__.py

```diff
@@ -0,0 +1,32 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+import azext_webapps._help  # pylint: disable=unused-import
+
+
+class WebappsCommandLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core.commands import CliCommandType
+        webapps_custom = CliCommandType(
+            operations_tmpl='azext_webapps.custom#{}')
+        super(WebappsCommandLoader, self).__init__(cli_ctx=cli_ctx,
+                                                   custom_command_type=webapps_custom)
```

Comment:
Add `min_profile` so this command is not available to Azure Stack users.
See: https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/__init__.py#L22